projects/ngx-tippy-wrapper/src/lib/ngx-tippy.directive.ts

```diff
@@ -25,6 +25,7 @@ export class NgxTippyDirective implements OnInit, OnDestroy {
   @Input() tippyProps?: NgxTippyProps;
   @Input() tippyName?: string;
   @Input() tippyClassName?: string;
+  @Input() tippyContext: any;
```

Comment:
`tippyContext` should be optional like previous attributes and I think it also should have some generic interface